app/renderer/js/pages/preference/base-section.js

```diff
@@ -19,14 +19,20 @@ class BaseSection extends BaseComponent {
 	generateOptionTemplate(settingOption) {
 		if (settingOption) {
 			return `
-				<div class="action green">
-					<span>On</span>
+				<div class="action">
+					<div class="switch">
+					  <input class="cmn-toggle cmn-toggle-round" type="checkbox" checked>
+					  <label for="cmn-toggle-1"></label>
+					</div>
 				</div>
 			`;
 		} else {
 			return `
-				<div class="action red">
-					<span>Off</span>
+				<div class="action">
+					<div class="switch">
+					  <input class="cmn-toggle cmn-toggle-round" type="checkbox">
+					  <label for="cmn-toggle-1"></label>
```

Comment:
I would use better class-names here. Changes this -
* `cmn-toggle cmn-toggle-round`
* `cmn-toggle-1`